File: MORM.CSharp/Models/Venda/Operacao.cs
using MORM.CSharp.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("OPERACAO")]
    public class Operacao : CollectionItem
    {
        [Campo("ID_OPERACAO", CampoTipo.Key)]
        public string Id_Operacao { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("DS_OPERACAO", CampoTipo.Req)]
        public string Ds_Operacao { get; set; }
        [Campo("TP_MODELONF", CampoTipo.Req)]
        public int Tp_Modelonf { get; set; }
        [Campo("TP_MODALIDADE", CampoTipo.Req)]
        public int Tp_Modalidade { get; set; }
        [Campo("TP_OPERACAO", CampoTipo.Req)]
        public int Tp_Operacao { get; set; }
        [Campo("CD_SERIE", CampoTipo.Req)]
        public string Cd_Serie { get; set; }
        [Campo("CD_CFOP", CampoTipo.Req)]
        public int Cd_Cfop { get; set; }
        [Campo("ID_REGRAFISCAL", CampoTipo.Req)]
        public int Id_Regrafiscal { get; set; }
    }
}

File: MORM.Java/src/MORM/Java/Models/Venda/Operacao.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "OPERACAO")
public class Operacao extends CollectionItem
{
    @CampoAttribute(Campo = "ID_OPERACAO", Tipo = CampoTipo.Key)
    public String Id_Operacao;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "DS_OPERACAO", Tipo = CampoTipo.Req)
    public String Ds_Operacao;
    @CampoAttribute(Campo = "TP_MODELONF", Tipo = CampoTipo.Req)
    public Integer Tp_Modelonf;
    @CampoAttribute(Campo = "TP_MODALIDADE", Tipo = CampoTipo.Req)
    public Integer Tp_Modalidade;
    @CampoAttribute(Campo = "TP_OPERACAO", Tipo = CampoTipo.Req)
    public Integer Tp_Operacao;
    @CampoAttribute(Campo = "CD_SERIE", Tipo = CampoTipo.Req)
    public String Cd_Serie;
    @CampoAttribute(Campo = "CD_CFOP", Tipo = CampoTipo.Req)
    public Integer Cd_Cfop;
    @CampoAttribute(Campo = "ID_REGRAFISCAL", Tipo = CampoTipo.Req)
    public Integer Id_Regrafiscal;
}